nautobot/extras/filters/__init__.py

```diff
@@ -249,6 +249,12 @@ class ConfigContextFilterSet(BaseFilterSet):
         to_field_name="model",
         label="Device Type (model or ID)",
     )
+    device_family = NaturalKeyOrPKMultipleChoiceFilter(
+        field_name="device_families",
+        queryset=DeviceFamily.objects.all(),
+        to_field_name="name",
+        label="Device Family (ID or name)",
```

Comment:
Just to match the other filters:

```suggestion
        label="Device Family (name or ID)",
```